### PR TITLE
Likes app

### PR DESCRIPTION
**Pull request by bgaultier**

### CODE REVIEW

File: apps/likes/laboite/apps/likes/models.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    page_name = models.CharField(_('Nom de la page'), max_length=96)
+    likes = models.PositiveIntegerField(_("Nombre de J'aime"), blank=True, null=True)
+
+    def _get_data(self):
+        return {'likes': self.likes}
+
+    def update_data(self):
+        url = "https://graph.facebook.com/v2.8/" + self.page_name + "?fields=fan_count&access_token=" + settings.FACEBOOK_ACCESS_TOKEN
+        r = requests.get(url)
+        print int(r.json().get("fan_count"))
```
Comment (vrialland) [inline apps/likes/laboite/apps/likes/models.py]: I think it was used for debug purpose, can you remove it?

File: apps/likes/laboite/apps/likes/models.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    page_name = models.CharField(_('Nom de la page'), max_length=96)
+    likes = models.PositiveIntegerField(_("Nombre de J'aime"), blank=True, null=True)
+
+    def _get_data(self):
+        return {'likes': self.likes}
+
+    def update_data(self):
+        url = "https://graph.facebook.com/v2.8/" + self.page_name + "?fields=fan_count&access_token=" + settings.FACEBOOK_ACCESS_TOKEN
+        r = requests.get(url)
```
Comment (vrialland) [inline apps/likes/laboite/apps/likes/models.py]: `requests.get` can use a `params` keyword than eases the readability eg:

```python
url = "https://graph.facebook.com/v2.8/" + self.page_name
params = {'fields': 'fan_count',
          'access_token': settings.FACEBOOK_ACCESS_TOKEN}
r = requests.get(url, params=params)
```

File: apps/likes/laboite/apps/likes/models.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+from django.db import models
+
+import requests
+
+from boites.models import App
+from . import settings
+
+
+class AppLikes(App):
+    UPDATE_INTERVAL = 1
```
Comment (vrialland) [inline apps/likes/laboite/apps/likes/models.py]: UPDATE_INTERVAL is by default in seconds, is it what you want? Or are you missing the unit like `1 * MINUTE`?

File: apps/likes/laboite/apps/likes/models.py (hunk 2 above)
Comment (bgaultier) [inline apps/likes/laboite/apps/likes/models.py]: Ooops, my bad :)

File: apps/likes/laboite/apps/likes/models.py (hunk 1 above)
Comment (bgaultier) [inline apps/likes/laboite/apps/likes/models.py]: sure :)

File: apps/likes/laboite/apps/likes/models.py (hunk 3 above)
Comment (vrialland) [inline apps/likes/laboite/apps/likes/models.py]: oops that's `MINUTES` with an "s" sorry

**Pull request merged**